pkgs/development/python-modules/slackclient/default.nix

```diff
@@ -19,14 +19,14 @@
 
 buildPythonPackage rec {
   pname = "python-slackclient";
-  version = "2.5.0";
+  version = "2.9.3";
 
   disabled = !isPy3k;
 
   src = fetchFromGitHub {
-    owner  = "slackapi";
-    repo   = pname;
-    rev    = version;
+    owner = "slackapi";
+    repo = pname;
+    rev = version;
     sha256 = "1ngj1mivbln19546195k400w9yaw69g0w6is7c75rqwyxr8wgzsk";
```

Comment:
no hash update?

Reply:
Thanks. I didn't notice that `nix-update` failed in the first place.

Reply:
Happens to me every once in a while as well, I'm not sure why that happens, but it should be investigated.